phpunit.xml.dist

```diff
@@ -44,12 +44,18 @@
 		<file>tests/Backend/Workspaces/WorkspacesUnloadTest.php</file>
 		<file>tests/Backend/Workspaces/WorkspacesLoadTest.php</file>
 		<file>tests/Backend/Workspaces/WorkspacesRenameLoadTest.php</file>
-		<directory>tests/Backend/Snowflake</directory>
-		<file>tests/Backend/Export/ExportParamsTest.php</file>
+		<file>tests/Backend/Snowflake/CloneIntoWorkspaceTest.php</file>
```

Comment:
Tyhle přibyly z toho directory, ale nejdou paralelně, je to tak?

Reply:
Jo přesně tak. Je to owrkspace a ty v prní fázi nejedou paralelně